### ISSUE TITLE
Aborting on shard error during drain in upgrade test

### ISSUE THREAD

**Issue by soyacz** [scylladb/scylladb#12365]
## Installation details

Kernel Version: 3.10.0-1160.80.1.el7.x86_64
Scylla version (or git commit hash): `5.0.6-20221114.b844d1482` with build-id `7489c4bb6d6a3880c4b77b72f5933d89a86ca6bb`

Cluster size: 4 nodes (n1-highmem-8)

Scylla Nodes used in this run:
- rolling-upgrade--centos-db-node-45240e15-0-4 (34.23.24.164 | 10.142.0.121) (shards: 8)
- rolling-upgrade--centos-db-node-45240e15-0-3 (35.231.246.238 | 10.142.0.104) (shards: 8)
- rolling-upgrade--centos-db-node-45240e15-0-2 (34.73.202.23 | 10.142.0.90) (shards: 8)
- rolling-upgrade--centos-db-node-45240e15-0-1 (34.73.86.185 | 10.142.0.68) (shards: 8)

OS / Image: `https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/centos-7` (gce: us-east1)

Test: `rolling-upgrade-centos7-test`
Test id: `45240e15-e91a-46ca-ae49-1fa4a872a16b`
Test name: `enterprise-2022.1/rolling-upgrade/rolling-upgrade-centos7-test`
Test config file(s):

- [rolling-upgrade.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/7d9fbadde7b6a3087117585b15b683ff2a4b6bb3/test-cases/upgrades/rolling-upgrade.yaml)

## Issue description

During upgrade from `5.0.6-20221114.b844d1482` to `2022.1.4-20221218.76bb50fe2`, after upgrade of 1 node and upgrade&rollback 
After steps:
1. upgrade 1 node
2. upgrade second node
3. rollback second node
4. verify data
during step 6.: upgrade rest of the nodes,
when draining a node before upgrade, scylla got into error state:
```
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]:  [shard 6] migration_manager - stopping migration service
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]:  [shard 0] seastar - Failed to allocate 43 bytes
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: Aborting on shard 0.
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: Backtrace:
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x42e75a8
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x43180f2
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x7fd536a86a1f
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: /opt/scylladb/libreloc/libc.so.6+0x3d2a1
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: /opt/scylladb/libreloc/libc.so.6+0x268a3
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x42bc3b5
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x42bef05
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x184d55e
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x1ad431e
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x19cef7d
Dec 18 08:18:20 rolling-upgrade--centos-db-node-45240e15-0-2 scylla[4052]: 0x4567f81
Dec 18 08:20:11 rolling-upgrade--centos-db-node-45240e15-0-2 systemd[1]: scylla-server.service: main process exited, code=dumped, status=6/ABRT
Dec 18 08:20:13 rolling-upgrade--centos-db-node-45240e15-0-2 systemd[1]: Unit scylla-server.service entered failed state.
```
SCT catched 'aborting on shard' error and produced event:
```
2022-12-18 08:18:20.678 <2022-12-18 08:18:20.000>: (DatabaseLogEvent Severity.ERROR) period_type=one-time event_id=ae98a67e-613c-4885-a79c-dd6a5125db10: type=ABORTING_ON_SHARD regex=Aborting on shard line_number=17696 node=rolling-upgrade--centos-db-node-45240e15-0-2
2022-12-18T08:18:20+00:00 rolling-upgrade--centos-db-node-45240e15-0-2 !    INFO | Aborting on shard 0.
seastar::basic_sstring<char, unsigned int, 15u, true>::is_internal() const at ./build/release/seastar/./seastar/include/seastar/core/sstring.hh:78
(inlined by) seastar::basic_sstring<char, unsigned int, 15u, true>::is_external() const at ./build/release/seastar/./seastar/include/seastar/core/sstring.hh:81
(inlined by) ~basic_sstring at ./build/release/seastar/./seastar/include/seastar/core/sstring.hh:202
(inlined by) ~label_instance at ./build/release/seastar/./seastar/include/seastar/core/metrics.hh:152
(inlined by) seastar::reactor::task_queue::register_stats() at ./build/release/seastar/./seastar/src/core/reactor.cc:834
seastar::future_state_base::any::move_it(seastar::future_state_base::any&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:481
(inlined by) seastar::future_state_base::any::operator=(seastar::future_state_base::any&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:499
(inlined by) seastar::future_state_base::operator=(seastar::future_state_base&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:539
(inlined by) seastar::future_state_base::set_exception(seastar::future_state_base&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:542
(inlined by) void seastar::internal::promise_base::set_exception_impl<seastar::future_state_base>(seastar::future_state_base&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:841
(inlined by) seastar::internal::promise_base::set_exception(seastar::future_state_base&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:856
(inlined by) operator() at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1573
(inlined by) seastar::continuation<seastar::internal::promise_base_with_type<unsigned long>, seastar::pollable_fd_state::sendto(seastar::socket_address, void const*, unsigned long)::$_13, seastar::future<unsigned long> seastar::future<void>::then_impl_nrvo<seastar::pollable_fd_state::sendto(seastar::socket_address, void const*, unsigned long)::$_13, seastar::future<unsigned long> >(seastar::pollable_fd_state::sendto(seastar::socket_address, void const*, unsigned long)::$_13&&)::{lambda(seastar::internal::promise_base_with_type<unsigned long>&&, seastar::pollable_fd_state::sendto(seastar::socket_address, void const*, unsigned long)::$_13&, seastar::future_state<seastar::internal::monostate>&&)#1}, void>::run_and_dispose() at ./build/release/seastar/./seastar/include/seastar/core/future.hh:767
?? ??:0
?? ??:0
?? ??:0
seastar::future_state_base::get_exception() && at ./build/release/seastar/./seastar/src/core/future.cc:175
(inlined by) seastar::make_nested(std::__exception_ptr::exception_ptr&&, seastar::future_state_base&&) at ./build/release/seastar/./seastar/src/core/future.cc:176
_ZN7seastar7promiseIvE13set_exceptionINS_13sleep_abortedEEENSt9enable_ifIXntsr3std7is_sameINSt16remove_referenceIT_E4typeENSt15__exception_ptr13exception_ptrEEE5valueEvE4typeEOS6_ at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1035
(inlined by) sleeper at ./build/release/seastar/./seastar/src/core/future-util.cc:115
seastar::internal::future_base::detach_promise() at ././seastar/include/seastar/core/future.hh:1170
(inlined by) seastar::internal::future_base::schedule(seastar::task*, seastar::future_state_base*) at ././seastar/include/seastar/core/future.hh:1175
(inlined by) seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::schedule(seastar::continuation_base<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >*) at ././seastar/include/seastar/core/future.hh:1372
(inlined by) void seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::schedule<seastar::internal::promise_base_with_type<sstables::foreign_sstable_open_info>, sstables::sstable::get_open_info() &::$_30, seastar::future<sstables::foreign_sstable_open_info> seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::then_impl_nrvo<sstables::sstable::get_open_info() &::$_30, seastar::future<sstables::foreign_sstable_open_info> >(sstables::sstable::get_open_info() &::$_30&&)::{lambda(seastar::internal::promise_base_with_type<sstables::foreign_sstable_open_info>&&, sstables::sstable::get_open_info() &::$_30&, seastar::future_state<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >&&)#1}>(sstables::sstable::get_open_info() &::$_30&&, sstables::sstable::get_open_info() &::$_30&&, seastar::future<sstables::foreign_sstable_open_info> seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::then_impl_nrvo<sstables::sstable::get_open_info() &::$_30, seastar::future<sstables::foreign_sstable_open_info> >(sstables::sstable::get_open_info() &::$_30&&)::{lambda(seastar::internal::promise_base_with_type<sstables::foreign_sstable_open_info>&&, sstables::sstable::get_open_info() &::$_30&, seastar::future_state<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >&&)#1}&&) at ././seastar/include/seastar/core/future.hh:1391
(inlined by) seastar::future<sstables::foreign_sstable_open_info> seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::then_impl_nrvo<sstables::sstable::get_open_info() &::$_30, seastar::future<sstables::foreign_sstable_open_info> >(sstables::sstable::get_open_info() &::$_30&&) at ././seastar/include/seastar/core/future.hh:1571
(inlined by) seastar::future<sstables::foreign_sstable_open_info> seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::then_impl<sstables::sstable::get_open_info() &::$_30, seastar::future<sstables::foreign_sstable_open_info> >(sstables::sstable::get_open_info() &::$_30&&) at ././seastar/include/seastar/core/future.hh:1605
(inlined by) seastar::internal::future_result<sstables::sstable::get_open_info() &::$_30, seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::future_type seastar::internal::call_then_impl<seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > > >::run<sstables::sstable::get_open_info() &::$_30>(seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >&, sstables::sstable::get_open_info() &::$_30&&) at ././seastar/include/seastar/core/future.hh:1234
(inlined by) seastar::future<sstables::foreign_sstable_open_info> seastar::future<seastar::foreign_ptr<seastar::lw_shared_ptr<sstables::shareable_components> > >::then<sstables::sstable::get_open_info() &::$_30, seastar::future<sstables::foreign_sstable_open_info> >(sstables::sstable::get_open_info() &::$_30&&) at ././seastar/include/seastar/core/future.hh:1520
(inlined by) sstables::sstable::get_open_info() & at ./sstables/sstables.cc:1417
managed_bytes at ././utils/managed_bytes.hh:189
(inlined by) compound_wrapper at ././keys.hh:147
(inlined by) prefix_compound_wrapper at ././keys.hh:572
(inlined by) clustering_key_prefix at ././keys.hh:817
(inlined by) void sstables::mc::writer::write_clustered<sstables::mc::rt_marker>(sstables::mc::rt_marker const&) at ./sstables/mx/writer.cc:714
void seastar::future_state<seastar::internal::monostate>::set<>() at ././seastar/include/seastar/core/future.hh:634
(inlined by) void seastar::internal::promise_base_with_type<void>::set_value<>() at ././seastar/include/seastar/core/future.hh:925
(inlined by) void seastar::futurize<void>::satisfy_with_result_of<seastar::future<void> seastar::future<void>::then_impl_nrvo<seastar::future<void> sstables::sstable::read_simple<(sstables::component_type)11, sstables::scylla_metadata>(sstables::scylla_metadata&, seastar::io_priority_class const&)::{lambda(seastar::file)#1}::operator()(seastar::file) const::{lambda(unsigned long)#1}::operator()(unsigned long) const::{lambda()#2}, seastar::future<void> >(seastar::future<void> sstables::sstable::read_simple<(sstables::component_type)11, sstables::scylla_metadata>(sstables::scylla_metadata&, seastar::io_priority_class const&)::{lambda(seastar::file)#1}::operator()(seastar::file) const::{lambda(unsigned long)#1}::operator()(unsigned long) const::{lambda()#2}&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, {lambda(unsigned long)#1}&, seastar::future_state<seastar::future<void> sstables::sstable::read_simple<(sstables::component_type)11, sstables::scylla_metadata>(sstables::scylla_metadata&, seastar::io_priority_class const&)::{lambda(seastar::file)#1}::operator()(seastar::file) const::{lambda(unsigned long)#1}::operator()(unsigned long) const::{lambda()#2}::monostate>&&)#1}::operator()(seastar::internal, seastar::internal::promise_base_with_type, seastar::future_state) const::{lambda()#1}>(seastar::internal, {lambda()#2}) at ././seastar/include/seastar/core/future.hh:2118
(inlined by) operator() at ././seastar/include/seastar/core/future.hh:1575
(inlined by) seastar::continuation<seastar::internal::promise_base_with_type<void>, seastar::future<void> sstables::sstable::read_simple<(sstables::component_type)11, sstables::scylla_metadata>(sstables::scylla_metadata&, seastar::io_priority_class const&)::{lambda(seastar::file)#1}::operator()(seastar::file) const::{lambda(unsigned long)#1}::operator()(unsigned long) const::{lambda()#2}, seastar::future<void> seastar::future<void>::then_impl_nrvo<{lambda(unsigned long)#1}, seastar::future<void> >({lambda(unsigned long)#1}&&)::{lambda(seastar::internal::promise_base_with_type<void>&&, {lambda(unsigned long)#1}&, seastar::future_state<seastar::internal::monostate>&&)#1}, void>::run_and_dispose() at ././seastar/include/seastar/core/future.hh:767
_ZN7seastar6futureINS_17smp_service_groupEE8scheduleINS_8internal22promise_base_with_typeIS1_EENS2_12finally_bodyIZZNS_14with_semaphoreINS_33named_semaphore_exception_factoryEZZNS_24create_smp_service_groupENS_24smp_service_group_configEENK3$_0clEvEUlvE_NS_12lowres_clockEEENS_8futurizeINSt13invoke_resultIT0_JEE4typeEE4typeERNS_15basic_semaphoreIT_T1_EEmOSG_ENUlSM_E_clINS_15semaphore_unitsIS9_SD_EEEEDaSM_EUlvE_Lb0EEEZNS2_17then_wrapped_nrvoIS2_SW_EENSE_ISM_E4typeESQ_EUlOS6_RSW_ONS_12future_stateIS1_EEE_EEvOSM_SQ_OSN_ at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1392
(inlined by) _ZN7seastar6futureINS_17smp_service_groupEE17then_wrapped_nrvoIS2_NS2_12finally_bodyIZZNS_14with_semaphoreINS_33named_semaphore_exception_factoryEZZNS_24create_smp_service_groupENS_24smp_service_group_configEENK3$_0clEvEUlvE_NS_12lowres_clockEEENS_8futurizeINSt13invoke_resultIT0_JEE4typeEE4typeERNS_15basic_semaphoreIT_T1_EEmOSD_ENUlSJ_E_clINS_15semaphore_unitsIS6_SA_EEEEDaSJ_EUlvE_Lb0EEEEENSB_ISJ_E4typeESN_ at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1666
(inlined by) seastar::lowres_clock<std::invoke_result>::type seastar::future<seastar::smp_service_group>::then_wrapped_common<false, seastar::future<seastar::smp_service_group>, seastar::future<seastar::smp_service_group>::finally_body<seastar::futurize<std::invoke_result<seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}>::type>::type seastar::with_semaphore<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}, seastar::lowres_clock>(seastar::basic_semaphore<seastar::named_semaphore_exception_factory, seastar::lowres_clock>&, unsigned long, std::invoke_result&&)::{lambda(seastar::basic_semaphore)#1}::operator()<seastar::semaphore_units<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}> >(auto, seastar::basic_semaphore)::{lambda()#1}, false> >(false&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1691
(inlined by) seastar::lowres_clock<std::invoke_result>::type seastar::future<seastar::smp_service_group>::then_wrapped_maybe_erase<false, seastar::future<seastar::smp_service_group>, seastar::future<seastar::smp_service_group>::finally_body<seastar::futurize<std::invoke_result<seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}>::type>::type seastar::with_semaphore<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}, seastar::lowres_clock>(seastar::basic_semaphore<seastar::named_semaphore_exception_factory, seastar::lowres_clock>&, unsigned long, std::invoke_result&&)::{lambda(seastar::basic_semaphore)#1}::operator()<seastar::semaphore_units<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}> >(auto, seastar::basic_semaphore)::{lambda()#1}, false> >(false&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1644
(inlined by) seastar::lowres_clock<std::invoke_result>::type seastar::future<seastar::smp_service_group>::then_wrapped<seastar::future<seastar::smp_service_group>::finally_body<seastar::futurize<std::invoke_result<seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}>::type>::type seastar::with_semaphore<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}, seastar::lowres_clock>(seastar::basic_semaphore<seastar::named_semaphore_exception_factory, seastar::lowres_clock>&, unsigned long, std::invoke_result&&)::{lambda(seastar::basic_semaphore)#1}::operator()<seastar::semaphore_units<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}> >(auto, seastar::basic_semaphore)::{lambda()#1}, false>, seastar::future<seastar::smp_service_group> >(seastar::basic_semaphore&&) & at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1628
(inlined by) seastar::future<seastar::smp_service_group> seastar::future<seastar::smp_service_group>::finally<seastar::futurize<std::invoke_result<seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}>::type>::type seastar::with_semaphore<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}, seastar::lowres_clock>(seastar::basic_semaphore<seastar::named_semaphore_exception_factory, seastar::lowres_clock>&, unsigned long, std::invoke_result&&)::{lambda(seastar::basic_semaphore)#1}::operator()<seastar::semaphore_units<seastar::named_semaphore_exception_factory, seastar::create_smp_service_group(seastar::smp_service_group_config)::$_0::operator()() const::{lambda()#1}> >(auto, seastar::basic_semaphore)::{lambda()#1}>(seastar::basic_semaphore&&) at ./build/release/seastar/./seastar/include/seastar/core/future.hh:1745
(inlined by) operator()<seastar::semaphore_units<seastar::named_semaphore_exception_factory, seastar::lowres_clock> > at ./build/release/seastar/./seastar/include/seastar/core/semaphore.hh:554
```
and coredump:
```corefile_url=https://storage.cloud.google.com/upload.scylladb.com/core.scylla.997.03026d1ad5cb4ca7ae8e5761198c18c0.4052.1671351500000000/core.scylla.997.03026d1ad5cb4ca7ae8e5761198c18c0.4052.1671351500000000.gz
backtrace=           PID: 4052 (scylla)
UID: 997 (scylla)
GID: 1002 (scylla)
Signal: 6 (ABRT)
Timestamp: Sun 2022-12-18 08:18:20 UTC (1min 33s ago)
Command Line: /usr/bin/scylla --blocked-reactor-notify-ms 500 --abort-on-lsa-bad-alloc 1 --abort-on-seastar-bad-alloc --abort-on-internal-error 1 --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix --num-io-queues=8 --io-properties-file=/etc/scylla.d/io_properties.yaml --lock-memory=1
Executable: /opt/scylladb/libexec/scylla;639ecb51
Control Group: /scylla.slice/scylla-server.slice/scylla-server.service
Unit: scylla-server.service
Slice: scylla-server.slice
Boot ID: 03026d1ad5cb4ca7ae8e5761198c18c0
Machine ID: d2f63d3a15ef770c1d769687106e90f0
Hostname: rolling-upgrade--centos-db-node-45240e15-0-2
Coredump: /var/lib/systemd/coredump/core.scylla.997.03026d1ad5cb4ca7ae8e5761198c18c0.4052.1671351500000000
Message: Process 4052 (scylla) of user 997 dumped core.
Stack trace of thread 4052:
#0  0x00007fd535ecb2a2 raise (libc.so.6)
#1  0x00007fd535eb4950 abort (libc.so.6)
#2  0x00000000042bc3b6 n/a (/opt/scylladb/libexec/scylla;639ecb51 (deleted))
#3  0x61737361632e6568 n/a (n/a)
```
download instructions:
```
download_instructions=gsutil cp gs://upload.scylladb.com/core.scylla.997.03026d1ad5cb4ca7ae8e5761198c18c0.4052.1671351500000000/core.scylla.997.03026d1ad5cb4ca7ae8e5761198c18c0.4052.1671351500000000.gz .
gunzip /var/lib/systemd/coredump/core.scylla.997.03026d1ad5cb4ca7ae8e5761198c18c0.4052.1671351500000000.gz
```

- Restore Monitor Stack command: `$ hydra investigate show-monitor 45240e15-e91a-46ca-ae49-1fa4a872a16b`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=45240e15-e91a-46ca-ae49-1fa4a872a16b)
- Show all stored logs command: `$ hydra investigate show-logs 45240e15-e91a-46ca-ae49-1fa4a872a16b`

## Logs:
- **db-cluster-45240e15.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/db-cluster-45240e15.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/db-cluster-45240e15.tar.gz)
- **monitor-set-45240e15.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/monitor-set-45240e15.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/monitor-set-45240e15.tar.gz)
- **loader-set-45240e15.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/loader-set-45240e15.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/loader-set-45240e15.tar.gz)
- **sct-runner-45240e15.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/sct-runner-45240e15.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/45240e15-e91a-46ca-ae49-1fa4a872a16b/20221218_084001/sct-runner-45240e15.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/enterprise-2022.1/job/rolling-upgrade/job/rolling-upgrade-centos7-test/41/)

**Comment by DoronArazii** [comment]
Looks similar to https://github.com/scylladb/scylladb/issues/10447

**Comment by avikivity** [comment]
The decoded trace looks bogus, did you use the right version?

**Comment by soyacz** [comment]
> The decoded trace looks bogus, did you use the right version?

Decoding was done automatically as part of SCT test. I tried to investigate why bogus decode could happen and I found that new base scylla version appeared in between test steps. It caused upgrade scylla from 5.0.6 to 5.0.7 during upgrade of system packages (which we do before upgrading scylla). It also means that Scylla binaries were upgraded when scylla was not stopped. So I guess this is the cause of the issue.

I created issue in SCT repo: https://github.com/scylladb/scylla-cluster-tests/issues/5738

I don't think we support such situations in upgrade - I propose to close this issue.
